pointillism-info.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index 565c78a..ce6160a 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -1,11 +1,18 @@
+ [Activity]
+ name = Pointillism
+-activity_version = 9
+-bundle_id = org.laptop.community.Pointillism
++activity_version = 9.1
++release_date = 2019-12-22
++metadata_license = CC0-1.0
++bundle_id = org.sugarlabs.Pointillism
+ exec = sugar-activity3 activity.Activity
+ show_launcher = yes
+ icon = activity
+ license = GPLv3+
+ summary = Create a pointillist image from the camera
++description = Pointillism is a Sugar activity to make photos in which small dots of color are applied in patterns to form an image.
+ repository = https://github.com/sugarlabs/pointillism.git
+-tags = Art
++tags = Graphics
++update_contact = tch@sugarlabs.org
++developer_name = Sugar Labs Community
++url = https://github.com/sugarlabs/pointillism-activity#pointillism
++screenshots = https://i.imgur.com/K4Y729f.png
```

Comment:
Please add another screenshot with the original webcam mode.

Reply:
@tchx84 I don't have one. I currently don't have my webcam drivers for arch

Reply:
@tchx84 If you prefer, I can add the images mentioned here https://github.com/tchx84/flathub/pull/3#discussion_r362470921

Reply:
There you go! don't forget to upload it safely as well, and run the validators too.

![Screenshot from 2020-01-02 12-17-58](https://user-images.githubusercontent.com/481418/71674476-0d336500-2d5a-11ea-95b6-8b0bbeea82f5.png)